### DISCUSSION TITLE
Providing GraalVM native image builds

### DISCUSSION THREAD

**Discussion by moppman** [AsamK/signal-cli#956] [General]
Dear AsamK,
would you consider providing native image binaries on new releases, using e.g. something like [this](https://github.com/marketplace/actions/github-action-for-graalvm)?
Thanks!

**Comment by AsamK** [comment]
Yeah, could be added to the release pipeline.

**Comment by KevinRoebert** [comment]
Addressed in https://github.com/AsamK/signal-cli/pull/970
